R/client.R

```diff
@@ -94,12 +94,16 @@ check_service <- function(request) {
   }
 
   if (httr::http_error(request)) {
-    usethis::ui_info("HTTP Status: {crayon::red(httr::http_status(request)$message)}")
-    usethis::ui_line()
+   ui_info(
+   	sprintf(
+   		"HTTP Status: %s",
+   		crayon::red(httr::http_status(request)$message)
+   		)
+   )
 
     is_monitor_up <- !is.null(curl::nslookup("monitor.emodnet.eu", error = FALSE))
     if (interactive() && is_monitor_up) {
-      if (usethis::ui_yeah("Browse the EMODnet OGC monitor?")) {
+      if (utils::askYesNo("Browse the EMODnet OGC monitor?")) {
```

Comment:
i think it's fine to have an easier interface since saying yes/no does not have very important consequences.
we might want to choose FALSE as a default though

Reply:
Yea if you cancel you get an error. How about?
```r
browse_monitor <- utils::askYesNo("Browse the EMODnet OGC monitor?", FALSE, prompts = "yes/no/cancel")
if(is.na(browse_monitor)) browse_monitor <- FALSE
if(browse_monitor){
```

Reply:
I made the change but why would an user choose to "cancel", what would they expect?